lib/node_modules/@stdlib/math/base/special/cosc/README.md

```diff
@@ -0,0 +1,102 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# cosc
+
+> Compute the derivative of the [cardinal sine][sinc] of a number.
+
+<section class="intro">
+
+The derivative of the normalized [cardinal sine][sinc] function is defined as
+
+` { cosc(x) = (cos( PI*x ) - sin( PI*x )/(PI*x)) / x } if x != 0, and 0 if x == 0 `
+
+
+<!-- </equation> -->
+
+for any real number `x`.
+
+</section>
+
+<!-- /.intro -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var cosc = require( '@stdlib/math/base/special/cosc' );
+```
+
+#### cosc( x )
+
+Computes the derivative of the normalized [cardinal sine][sinc] of a `number`.
+
+```javascript
+var v = cosc( 0.5 );
+
+// Return ~-1.273
+
+var x = cosc( -1.2 );
```

Comment:
```suggestion
v = cosc( -1.2 );
```

It is okay to reassign to variables within an example.

Reply:
Fixed (the linter was killing me here but now it didn't seem to mind anymore :))